doc/source/parameters/cfd/volume_of_fluid.rst

```diff
@@ -89,7 +90,9 @@ The default values of the VOF parameters are given in the text box below.
 
 	Applying viscous dissipation in one of the fluids instead of both is particularly useful when one of the fluids is air. For numerical stability, the ``kinematic viscosity`` of the air is usually increased. However, but we do not want to have viscous dissipation in the air, because it would result in an unrealistic increase in its temperature.
 
-* ``subsection interface sharpening``: defines parameters to counter numerical diffusion of the VOF method and to avoid the interface between the two fluids becoming more and more blurry after each time step.
+  * ``limit force`` enables limiting the source term (for instance gravity force).
```

Comment:
```suggestion
  * ``limit force``: defines if the ``source term`` (doc:`../source_term`, classically the gravity force) should be applied to the non-conservative fluid. This parameter is used if ``conservative fluid`` is not ``both`` in ``subsection mass conservation``.     
```

Reply:
Let's leave it this way for now, but I am really, really starting to hate this multiplication of parameters. I would rather remove it alltogether and apply the force everywhere. The VOF models already has a bit too many shenigans right now.

Reply:
It is removed.